nucliadb_utils/nucliadb_utils/nats/connection_manager.py

```diff
@@ -207,6 +172,7 @@ async def subscribe(
         cb: Callable[[Msg], Awaitable[None]],
         subscription_lost_cb: Callable[[], Awaitable[None]],
         flow_control: bool = False,
+        manual_ack: bool = False,
```

Comment:
hmm, we're manually acking everywhere right now I think. I didn't know about this control

Reply:
Dang, I didn't know this! We should use this setting everywhere else as well.